docs/testplan/BGP-Route-Aggregation.md

```diff
@@ -0,0 +1,54 @@
+- [Overview](#overview)
+    - [Scope](#scope)
+    - [Testbed](#testbed)
+- [Setup configuration](#setup-configuration)
+- [Test cases](#test-cases)
+
+# Test name
+
+BGP Route Aggregation
+
+## Overview
+
+The goal of this test is to verify that the BGP route aggregation behaves as expected.
+
+### Scope
+
+The test is targeting a running SONIC system with fully functioning configuration. The purpose of the test is to test for BGP route aggregation.
+
+### Related DUT CLI commands
+
+| Command | Comment |
+| ------- | ------- |
+|Configuration commands|
+| clear bgp * soft | Soft reset of BGP neighbors and routes |
+|Show commands|
+| show ip bgp summary | Dispaly current memory statistics, can be done with ipv6 too |
+
+### Related DUT configuration files
+
+N/A
+
+### Related SAI APIs
+
+N/A
+
+## Test structure
+### Setup configuration
+
+This test requires BGP neighbors to be configured and established.
+
+### Configuration scripts
+
+N/A
+
+## Test cases
+### Test case #1 - Route Aggregation
+
+#### Test objective
+Aggregate BGP routes in various methods
+1. Gather base BGP information from DUT
+2. Configure summary only route aggretation
+3. Verify aggregate route is advertised
+4. Configure as-set route aggregation
+5. Verify aggregate route is advertised
```

Comment:
The test plan looks generic for the route aggregation, but only a limited options are verified.
There are other combinations which are applicable to route-aggregation, e.g.
- aggregation without suppressing the specific routes
- aggregation with route-map
- aggregation with suppress-map